src/Microsoft.DotNet.Tools.Builder/CompileContext.cs

```diff
@@ -148,14 +148,14 @@ private static bool AnyMissingIO(ProjectContext project, IEnumerable<string> ite
                 return false;
             }
 
-            Reporter.Verbose.WriteLine($"\nProject {project.ProjectName()} will be compiled because expected {itemsType} are missing: ");
+            Reporter.Output.WriteLine($"\nProject {project.ProjectName()} will be compiled because expected {itemsType} are missing: ");
 
             foreach (var missing in missingItems)
```

Comment:
In non-verbose output, I can output for each project:

- if incremental, say something like "Project Foo was previously compiled. Skipping compilation."
- if not incremental, say something like "Project Foo will be compiled. More details on verbose log."